### ISSUE TITLE
accessibility: there should be a way to view a history of alerts

### ISSUE THREAD

**Issue by hiddenist** [Spectrologer/Material-Tower-Defense#32]
Not everybody can read quickly, and even if you increase the time messages are listed for there's no perfect duration for every person. Also, even those that can read quickly may forget what a message said.

Adding a history with a log from the current game's alerts would help if someone wanted  to look at messages longer or review past alerts.
